script/foreman-debug

```diff
@@ -349,8 +349,7 @@ add_files /var/log/foreman/apipie_cache*.log*
 add_files /var/log/foreman/cron*.log*
 add_files /var/log/foreman/db_migrate*.log*
 add_files /var/log/foreman/db_seed*.log*
-add_files /var/log/foreman/production-*.log*
-add_files /var/log/foreman/production.log
+add_files /var/log/foreman/production.log*
```

Comment:
Please keep them separate, it was on purpose - the function have some limits so we want to make sure production.log goes in with its own limits. So simply just fix the line 352.